### DISCUSSION TITLE
Bookmarked radio stations cannot be played

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1762] [Fixed Issues] [answered]
RPI 3B, Updated to 20240818

If I choose radio stations from the Web Radio Library page I can add or replace them for Playback quite happily using the Add/Play and Replace/Play menu items. However if I Bookmark the same stations they appear on the Library main page (with their logos) but the Add/Play and Replace/Play options on the Bookmark do nothing at all. 

Interestingly, using 'journalctl -f' from an SSH command line shows nothing happening either.  This usually shows some sort of activity when changing pages/options etc.

I suspect this isn't a new problem and may have existed for a while.

Chris

**Comment by ceejayemm** [comment]
If I clear the Playlist and the use the Add/Play option this seems to work but Replace/Play still has no effect.

Chris

**Comment by ceejayemm** [comment]
@rern 
Any chance this might get fixed soon ?

Thanks

Chris

**Reply by rern** [reply, marked as answer]
Try the latest UPDATE.

**Reply by ceejayemm** [reply]
@rern 
Thanks for the quick response.  The latest Update branch seems to fix this problem.

Chris

**Comment by ceejayemm** [comment]
Confirmed fixed in 20240921.

Many thanks 

Chris
